packages/backend-function/src/resolve_secret_banner.ts

```diff
@@ -1,37 +1,53 @@
 /* The code in this line replaces placeholder text in environment variables for secrets with values fetched from SSM, this is a noop if there are no secrets */
 import { SSM } from '@aws-sdk/client-ssm';
+import type { AmplifySecretPaths } from './function_env_translator.js';
```

Comment:
won't this cause problems ? 
importing from SDK above is fine that is expected to be present in lambdas but I'd expect that import from local path is going to fail... unless bundler can resolve this somehow ? - could you share how bundled code looks like ?

Reply:
`import type` is completely erased from transpiled code. It's just for the TS type engine

Reply:
That's what I'd expect to happen during normal bundling but I'm not sure how this works for "banner"

Reply:
Confirmed just now that "banner" doesn't include `import type` as well looking at the console